README.md

```diff
@@ -1 +1,3 @@
 # test-repo
+
+This repo for learning purpose
```

Comment:
please correct the repo meaning

Reply:
repo is short name so it is correct please merge it